docs/use/glue.md

```diff
@@ -316,20 +317,58 @@ Which we reference as Equation {eq}`eq-sym`.
 `glue:math` only works with glued variables that contain a `text/latex` output.
 ```
 
+### The `glue:md` role/directive
+
+With `glue:md`, you can output `text/markdown`, that will be integrated into your page.
+
+````{code-cell} ipython3
+from IPython.display import Markdown
+glue("inline_md", Markdown(
+  "inline **markdown** with a [link](glue/main), "
+  "and a nested glue value: {glue:}`boot_mean`"
+), display=False)
+glue("block_md", Markdown("""
+#### A heading
+
+Then some text, and anything nested.
+
+```python
+print("Hello world!")
+```
+"""
+), display=False)
+````
+
+The format of the markdown can be specified as:
+
+- `commonmark` (default): Restricted to the [CommonMark specification](https://commonmark.org/).
+- `gfm`: Restricted to the [GitHub-flavored markdown](https://github.github.com/gfm/).
+  - Note, this requires the installation of the [linkify-it-py package](https://pypi.org/project/linkify-it-py)
+- `myst`: The MyST parser configuration for the the current document.
+
+For example, the following role/directive will glue inline/block MyST Markdown, as if it was part of the original document.
+
+````md
+Here is some {glue:md}`inline_md:myst`!
+
+```{glue:md} block_md
+:format: myst
+```
+````
+
+Here is some {glue:md}`inline_md:myst`!
+
+```{glue:md} block_md
+:format: myst
+```
+
 +++
 
 ## Advanced glue usecases
 
 Here are a few more specific and advanced uses of the `glue` submodule.
 
-### Pasting from pages you don't include in the documentation
-
-Sometimes you'd like to use variables from notebooks that are not meant to be
-shown to users. In this case, you should bundle the notebook with the rest of your
-content pages, but include `orphan:` in the metadata of the notebook.
-
-For example, the following text: `` {glue:}`orphaned_var` was created in {ref}`orphaned-nb` ``.
-Results in: {glue:}`orphaned_var` was created in {ref}`orphaned-nb`
+### Pasting
```

Comment:
I think either this needs to be bumped down to `##` or the subsequent headers should be bumped up to `####`

Reply:
I just removed that heading entirely